ntc_templates/templates/cisco_ios_show_tacacs.textfsm

```diff
@@ -24,6 +24,8 @@ Start
   ^\s+Failed\s+Connect\s+Attempts:\s+${FAILED_CONNECTIONS}\s*$$
   ^\s+Total\s+Packets\s+Sent:\s+${PACKETS_SENT}\s*$$
   ^\s+Total\s+Packets\s+Recv:\s+${PACKET_RECEIVED}\s*$$
+  ^\s+Server\s+Status:\s+\S+\s*$$
+  ^Continous\s+Authc\s+fail\s+count:\s+\d+\s*$$
```

Comment:
This is a new one - just in case there's ever white space in front of the Continuous lines.

```suggestion
  ^\s*Continous\s+Authc\s+fail\s+count:\s+\d+\s*$$
```

Reply:
I was too slow on the previous one and you had already accepted and resolved it.